core/src/main/java/io/jenkins/pluginhealth/scoring/probes/RenovateProbe.java

```diff
@@ -0,0 +1,25 @@
+package io.jenkins.pluginhealth.scoring.probes;
+
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+@Component
+@Order(RenovateProbe.ORDER)
+public class RenovateProbe extends AbstractDetectBotConfigurationProbe {
+    public static final int ORDER = LastCommitDateProbe.ORDER + 100;
+    public static final String KEY = "renovate";
+
+    RenovateProbe() {
+        super("renovate");
```

Comment:
Why not using `KEY` here?

Reply:
same as https://github.com/jenkins-infra/plugin-health-scoring/pull/350/files#r1265473400